README.md

```diff
@@ -290,18 +290,26 @@ for a `.pip-tools.toml` file and then in your `pyproject.toml`. You can
 also specify an alternate TOML configuration file with the `--config` option.
 
 For example, to by default generate `pip` hashes in the resulting
-requirements file output, you can specify in a configuration file
+requirements file output, you can specify in a configuration file:
 
 ```toml
 [tool.pip-tools]
 generate-hashes = true
-
 ```
 
 Options to `pip-compile` and `pip-sync` that may be used more than once
 must be defined as lists in a configuration file, even if they only have one
 value.
 
+`pip-tools` supports default values vor [all valid command-line flags](/cli/index.md)
+of its subcommands. Also, it is possible to specify configuration keys in
+command-line manner, for example:
+
+```toml
+[tool.pip-tools]
+--generate-hashes = true
+```
```

Comment:
Perhaps my [comment](https://github.com/jazzband/pip-tools/pull/1913#discussion_r1272691430) was confusing, I apologise. I meant _options that represented as a snake/kebab case string **without double dash prefix**_. For example:

```toml
# kebab and snake case
generate-hashes = true
generate_hashes = false

# booleans
annotate = false
no-annotate = true

# ... or inverted booleans
annotate = true
no-annotate = false 

```